habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -104,7 +104,6 @@ class CenterCropperConfig(ObsTransformConfig):
 
 
 cs.store(
-    package="habitat_baselines.rl.policy.obs_transforms.center_cropper",
```

Comment:
Why did you remove all of the packages here?

Reply:
It is needed because now obs_transforms need to be accessed as:

habitat_baselines.rl.policy.**agent_name**.obs_transforms.center_cropper